packages/waku/src/lib/handlers/handler-dev.ts

```diff
@@ -210,8 +210,6 @@ export function createHandler<
             .pipeTo(res.stream);
           return;
         }
-        next();
-        return;
```

Comment:
This is an intentional change in #510.
If this is causing some issues, I think `willBeHandledByVite` isn't working as expected.

Reply:
It was intentional at that point, but this prevents skipping SSR conditionally even with `--with-ssr`, so I need to do this anyway.